apps/acceptance/lib/view.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var Marionette = require('marionette-client');
+
+function View(client) {
+    this.client = client.scope({searchTimeout: 5000});
+    this.actions = new Marionette.Actions(this.client);
+}
+
+View.prototype = {
+    //Get the root element of the app and return it.
+    getRootElement: function (rootSelector) {
+        return this.client.findElement(rootSelector);
+    },
+    //From the root, find a child element based on selector.
+    findElement: function (selectorObject) {
+        return this.getRootElement(selectorObject.root).findElement(selectorObject.element);
```

Comment:
I think this is a huge constraint to, each time, pass the root of the View. It'll lead to a lot of duplication, I think. Let's assume the selector is defined in the Child class, like it's done in the [Calendar view](https://github.com/mozilla-b2g/gaia/blob/master/apps/calendar/test/marionette/lib/views/view.js#L31). In our case here, I would just do:

``` javascript
 findElement: function (selector) {
    return this.getRootElement().findElement(selector);
}

function getRootElement() {
    return this.client.findElement(this._SELECTORS.rootElement);
}
```